.gitignore

```diff
@@ -57,9 +57,6 @@ typings/
 # dotenv environment variables file
 .env
 
-# build
-build/**
```

Comment:
Can you put this back. We don't need the build assets to be included with the repo.

Reply:
Yep sorry @rizowski I needed to commit it only to check, since fetching my fork from github was without `build` 😄